incubator/redis-cache/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for redis-cache.
```

Comment:
Have you seen my comment regarding structural changes regarding the configuration? Please group by component, e. g.:

```yaml
redis:
  image:
    repository: 
    tag: 
    pullPolicy:
  service:
  resources:
microSentinel:
  image:
    repository: 
    tag: 
    pullPolicy:
  resources:
...
```